quickstart/renditions.md

```diff
@@ -1,59 +1,33 @@
-## Generate Renditions (Only Fullsize/2560) for Print Workflow 
+## Read and Generate Renditions
 
-Generate renditions for an original file asynchronously. Allowed rendition types are fullsize and 2560. Generated rendition will be deleted after 1 day automatically.
+##Renditions
+Each photo asset has a set of JPEGs intended for application display of the latest visual representation of the asset (either edited or original uploaded state)
 
+##Details of Renditions
 
-##Workflow to generate renditions
+* Creation - Upon uploading a photo, the following resources will be created (JPEG format)
+    * thumbnail2x (320 pixels constraint on either edge) 
+    * 640 (640 pixels on long edge)
+    * 1280 (1280 pixels on long edge) 
+    * 2048 (2048 pixels on long edge) 
+    
+* Edit - Upon editing a photo, the following resources will be newly created with all edits applied
+    * thumbnail2x (320 pixels constraint on either edge) 
+    * 640 (640 pixels on long edge) 
+    * 1280 (1280 pixels on long edge) 
+    * 2048 (2048 pixels on long edge) 
+    
+For best performance, use the smallest size possible for all application scrolling case 
 
-_STEP 1_: Create an asset
 
-```
-PUT /v2/catalogs/{catalog_id}/assets/{asset_id} HTTP/1.1 
-Authorization: {auth_token}
-Content-Type: application/json
-Content-Length: {xsd:nonNegativeInteger}
-If-None-Match: {xsd:string}
-{
-	"subtype": "{asset_subtype}",
-	"payload": {
-		"captureDate": "0000-00-00T00:00:00",
-		"importSource": {
-			"fileName": "{file_name}",
-			"importedOnDevice": "{import_device_name}",
-			"importedBy": "{import_account_id}",
-			"importTimestamp": "{import_time}"
-		}
-	}
-}
-```
-
-Sample success response:
-
-```
-HTTP/1.1 201 
-CreatedContent-Length: 0
-Location: {xsd:anyURI}
-```
-
-_STEP 2_: Upload original for an asset.
-
-```
-PUT /v2/catalogs/{catalog_id}/assets/{asset_id}/master HTTP/1.1
-Authorization: {auth_token}
-Content-Length: {xsd:nonNegativeInteger}
-Content-Range: {xsd:string}
-Content-Type: {xsd:string}
-```
+##Workflow to generate renditions for print workflow
 
-Sample success response:
+Generate renditions for an original file asynchronously. Allowed rendition types are fullsize and 2560. Generated rendition will be deleted after 1 day automatically. Both these renditions must always be requested on demand.  First check to see if one is available, and if not create it. These renditions are not available with basic API access. Should your application require this capability, you must contact Adobe (lightroomcontact@adobe.com) and describe your needs. Here are the details of fullsize and 2560
```

Comment:
Should remove "(lightroomcontact@adobe.com)".

Reply:
@mpeercy Is it fine to include to just contact Adobe and not mention any email address for now? Folks need to raise issue in order to contact us then.